plugins/com.act21.mamba.layout.alg.tree/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
```

Comment:
Compiler compliance is currently 1.8 (I know it also works for 11) but please do not change such things. I suggest a separate PR for this.